pkg/cmd/server/origin/admission/register.go

```diff
@@ -117,24 +117,38 @@ var (
 	)
 )
 
-func init() {
-	admission.PluginEnabledFn = IsAdmissionPluginActivated
-}
-
-func IsAdmissionPluginActivated(name string, config io.Reader) bool {
+func IsAdmissionPluginActivated(name string, config io.Reader) (bool, bool) {
```

Comment:
Thsi looks really similar to the upstream method now that we've pushed changes upstream.  Any chance we can do a more invasive refactor to figure out what we want?